definition-provider-plugin/package.json

```diff
@@ -0,0 +1,39 @@
+{
+      "name": "definition-provider-plugin-plugin",
+      "publisher": "theia",
+      "keywords": [
+        "theia-plugin"
+      ],
+      "version": "0.0.1",
+      "license": "[object Object]",
```

Comment:
I think there is an issue there on the license

Reply:
Seems so...